lib/common/core/CmdParser.cpp

```diff
@@ -591,10 +597,7 @@ int CmdLineArgsParser::Parse(__in LPWSTR oneArg) throw()
                 PrintUsage();
                 return -1;
             }
-            else
-            {
-                ParseFlag();
-            }
+            ParseFlag();
```

Comment:
I know it's equivalent but why get rid of the else block wrapping this statement?